modules/social_features/social_event/modules/social_event_enrollment_max/social_event_max_enroll.module

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @file
+ * The Social Event Max Enroll module.
+ */
+
+use Drupal\views\ViewExecutable;
+use Drupal\views\Plugin\views\cache\CachePluginBase;
+use Drupal\node\Entity\Node;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_views_post_render().
+ *
+ * Alter "Event enrollments" views. Add number of places left .
+ */
+function social_event_max_enroll_views_post_render(ViewExecutable $view, &$output, CachePluginBase $cache) {
```

Comment:
This doesn't seem to work for me.

Reply:
@bramtenhove What do you mean that it doesn't work?
This function adds text **# places left** to enrollments block. You can see it on screenshots here: https://jira.goalgorilla.com/browse/ECI-851

Reply:
That hook didn't fire for me for unknown reason. Perhaps I did something wrong though, the screenshots indeed proof it works, just not for me :(